web/public/js/services/dojo-utils.js

```diff
@@ -95,5 +95,40 @@ angular.module('cpZenPlatform').factory('dojoUtils', ['$location', '$translate',
     }
     return deferred.promise;
   }
+
+  dojoUtils.getFrequencyStrings = function () {
+    var strings = {};
+    strings.frequencies = [
+      { id: '2/w',
+      name: $translate.instant('Twice Weekly')},
+      { id: '1/w',
+      name: $translate.instant('Weekly')},
+      { id: '2/m',
+      name: $translate.instant('Bi-weekly/Fortnightly/Every two weeks')},
+      { id: '1/m',
+      name: $translate.instant('Monthly')},
+      { id: 'other',
+      name: $translate.instant('Other')}
+    ];
+    strings.monthlyFrequencies = [
+      {id: 'first', name: $translate.instant('First')},
+      {id: '2nd', name: $translate.instant('Second')},
+      {id: '3rd', name: $translate.instant('Third')},
+      {id: 'last', name: $translate.instant('Last')}
```

Comment:
My point is that having a 5th week is so "unexpected" that you plan on the scenario that is consistent